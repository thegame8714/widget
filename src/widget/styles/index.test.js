import React from 'react'
import { Title } from './'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

describe('Style components', () => {
  describe('Title', () => {
    it('should render the Title component', () => {
      const titleStyledComponent = renderer.create(<Title />).toJSON()
      expect(titleStyledComponent).toMatchSnapshot()
    })
  })
})
