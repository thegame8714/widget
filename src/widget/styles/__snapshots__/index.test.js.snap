// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Style components Title should render the Title component 1`] = `
.c0 {
  color: #550000;
}

<h1
  className="c0"
/>
`;
